spec/requests/prosecution_cases_spec.rb

```diff
@@ -22,7 +22,6 @@
         get "/prosecutionCases?prosecutionCaseReference=incorrect-reference", headers: headers
         expect(response).to have_http_status(:ok)
         expect(response.headers["content-type"]).to eq("application/vnd.unifiedsearch.query.laa.cases+json")
-        expect(response.body).to match_json_schema(:search_prosecution_case_response)
```

Comment:
If this is no longer relevant, can we also change the name of the test on line 21 please?